examples/mnist.py

```diff
@@ -23,15 +33,12 @@
 
 
 def _load_data(url=DATA_URL, filename=DATA_FILENAME):
-    try:
-        with gzip.open(filename, 'rb') as f:
-            data = pickle.load(f)
-    except (IOError, EOFError):
+    if not os.path.exists(filename):
```

Comment:
>Not sure "most users" won't understand what a checksum error is and a traceback that points right at the line that opens the file in question. I guess we might be underestimating our users. :-)

Just don't be surprised when it pops up on the mailing list or SO. ;)

> And since you added that print("Downloading MNIST") line I do hope that users are less inclined now to kill the process in the middle.

That's what I was hoping.  That's why I killed the process in the first place! :)